src/stories/components/custom-table/row-item/item-core-unit.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { columns, CoreUnit } from '@ses/core/utils/test.utils';
+import ItemCoreUnit from './item-core-unit';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/CUTable/ItemCoreUnit',
+  component: ItemCoreUnit,
+  parameters: {
+    fullscreen: 'centered',
+    chromatic: {
+      viewports: [1194, 1280],
+      pauseAnimationAtEnd: true,
+    },
+  },
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          margin: '3em',
+          width: 'fit-content',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as ComponentMeta<typeof ItemCoreUnit>;
+
+const variantsArgs = [
+  {
+    queryStrings: '',
+    key: '2',
+    loading: false,
+    columns,
+    cu: CoreUnit,
+  },
+];
+export const [[Summary, SummaryDark]] = createThemeModeVariants(ItemCoreUnit, variantsArgs);
```

Comment:
Please, use other names according to the component name, as this is not related with summary and can be confusing

Reply:
done